### ISSUE TITLE
Splitting problems

### ISSUE THREAD

**Issue by PerritoCuleado** [OgarProject/Ogar#306]
The original agar.io has a feature that it's activated when you split, it last just a second, at that moment the cells are capable of going through other cells, you can even see them coming out of your cell when you hit space and divide, once that feature its inactivated and your cells go back to its normal speed, the ones that are still inside are pulled to the outside quickly, almost immediately  to the outside, sadly Ogar doesn't have that feature, therefore, some common troubles happen:
-If you have tiny and big cells at the same time and you split, the ones that are are being thrown (usually the big ones) will hit the tiny ones and stop middle way
-if your target is a little bit inside one of your cells, like two-thirds of its circle, and you know that you're big enough to eat it, you'll hit space, but your cell won't eat him, at that moment It appears that you went through him, but what really happened was that your cell did not travel in a trajectory, it just ''appeared' where the ''space speed' was supposed to end.

Also, the original agar.io has another feature that led the cells to go a lite bit inside each other when they're moving, that makes them a little bit less wobbly, and makes teaming and the action of giving mass easier, they won't shake around and potentially eat your teammate.

**Comment by NatsuTheGreat** [comment]
So what's your point? xP

You can the code to this feature?

**Comment by PerritoCuleado** [comment]
Nope, i dont know how to code that feature, also splitting speed is another problem,

**Comment by mrmarktyy** [comment]
I see original agar.io changes the effect of splitting cells all the time (speed, collision, etc)
I don't see there's a way to just blindly tuning ogar to be identical to agar.io
but, of course, with some efforts, we can make ogar close to agar.io

**Comment by ItzLevvie** [comment]
Close? @vram4
